packages/pulpcore/python-pulp-deb/python-pulp-deb.spec

```diff
@@ -0,0 +1,56 @@
+# Created by pyp2rpm-3.3.2
+%global pypi_name pulp-deb
+
+Name:           python-%{pypi_name}
+Version:        2.3.0b1
+Release:        1%{?dist}
+Summary:        pulp-deb plugin for the Pulp Project
+
+License:        GPLv2+
+URL:            https://pulpproject.org/#deb
+Source0:        https://files.pythonhosted.org/packages/source/p/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
+BuildArch:      noarch
+
+BuildRequires:  python3-devel
+BuildRequires:  python3-pulpcore < 3.4
+BuildRequires:  python3-pulpcore >= 3.3
+BuildRequires:  python3-debian >= 0.1.36
```

Comment:
if you drop these buildrequires, the package will build fine without spliting the PR into two ;)

Reply:
done